swift/extractor/SwiftTagTraits.h

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include <swift/AST/ASTVisitor.h>
+
+// autogenerated header
+#include "swift/extractor/trap/TrapTags.h"
```

Comment:
temporarily at this moment (modulo some changes in naming and namespacing), you can see the file [here](https://github.com/github/semmle-code/blob/redsun82/swift-codegen/language-packs/swift/include/extractor/trap/generated/Tags.h). More long-term, it could be a good idea to make those generated files artifacts of the `swift/codegen` action and provide a link here maybe?

Reply:
The file is not present next to all the other header files, so my intention here was to say it's autogenerated to eliminate the confusion (e.g. "how does it compile if the file is not there????").

The files are generated by [`codegen`](https://github.com/github/codeql/pull/9034)

Regarding the contents, I'm only aware of this method:

```bash
> find `bazel info output_base` | grep TrapTags.h
```
@redsun82 is there a better way?

Reply:
@redsun82 as a side-note: we should put the header under additional `generated` (or similar) to eliminate the need for the `autogenerated header` comment.

Reply:
yeah, good idea to add `generated` to the path.

What I do, is to just ctrl-click the header and I get to it, but that requires the IDE to know where to look. If you are not using the bazel plugin because of the breakpoint issue, then you need to instruct somehow else the header search path (we only have one directory with generated headers)

Reply:
I think @sashabu's point though is to be able to read this code without even starting a build (for example, browsing through github, or reading a PR without pulling it in)

Reply:
I like not having generated code changes in the PR UI, but sometimes a reviewer might need to access that

Reply:
you can now find the generated artifacts in the action run summary of a PR (for example https://github.com/github/codeql/actions/runs/2319256039 ). However in order to make a link that we insert in code we would require more effort